src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -714,6 +718,30 @@ def apply_stage_change(self, context: PhaseContext):
         context.criterion.use_l1 = True
 
 
+@register_callback(Callbacks.ROBOFLOW_RESULT_CALLBACK)
+class RoboflowResultCallback(Callback):
+    """Append the training results to a csv file. Be aware that this does not fully overwrite the existing file, just appends."""
+
+    def __init__(self, dataset_name: str, output_path: Optional[str] = None):
+        """
+        :param dataset_name:    Name of the dataset that was used to train the model.
+        :param output_path:     Full path to the output csv file. By default, save at 'checkpoint_dir/results.csv'
+        """
+        self.dataset_name = dataset_name
+        self.output_path = output_path or os.path.join(get_project_checkpoints_dir_path(), "results.csv")
```

Comment:
The code should respect the case when `get_project_checkpoints_dir_path` returns None (We had this issue in Docker for out-of-source run)

Reply:
Same as previous comment, do we want to keep this?
If we don't use/want to use it might be better to take it out.